src/core/identity/identity.js

```diff
@@ -43,19 +43,29 @@ export const IDENTITY_PREFIX: NodeAddressT = NodeAddress.fromParts([
   "IDENTITY",
 ]);
 
-export function newIdentity(subtype: IdentityType, name: string): Identity {
+export function newIdentity(
+  subtype: IdentityType,
+  name: string,
+  aliases?: $ReadOnlyArray<Alias>
```

Comment:
Instead of having two different variables (aliases + actualAliases), can just set default value here:
```suggestion
  aliases: $ReadOnlyArray<Alias> = []
```